integration_tests/src/main/python/fastparquet_compatibility_test.py

```diff
@@ -167,6 +203,8 @@ def test_reading_file_written_with_gpu(spark_tmp_path, column_gen):
     There are xfails here because of fastparquet limitations in handling Decimal, Timestamps, Dates, etc.
     """
     data_path = spark_tmp_path + "/FASTPARQUET_TEST_GPU_WRITE_PATH"
+    local_base_path = (spark_tmp_path + "_local")
```

Comment:
I think this is okay, but it does assume that the temp path in HDFS is going to be available in the local file system too. I don't think there is anything we can do about that though.